pkg/virt-handler/vm.go

```diff
@@ -1943,8 +1943,11 @@ func lookupVMIPodNetworkName(networks []v1.Network) string {
 
 func (d *VirtualMachineController) validateSRIOVInterfacesForMigration(vmi *v1.VirtualMachineInstance) error {
 	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
-		if iface.SRIOV != nil && !d.clusterConfig.SRIOVLiveMigrationEnabled() {
-			return fmt.Errorf("SRIOVLiveMigration feature-gate is closed, can't migrate VMI with SRIOV interfaces")
+		if iface.SRIOV != nil {
+			if !d.clusterConfig.SRIOVLiveMigrationEnabled() {
+				return fmt.Errorf("SRIOV Live Migration enabled, virtual machine with SRIOV interfaces live migration is not supported yet")
```

Comment:
I think they need to be formatted similarly.
How about:
`SR-IOV Live Migration enabled, however live migration of guest with SR-IOV interfaces is not supported yet`

And the second:
`SR-IOV Live Migration disabled, live migration of guest with SR-IOV interfaces is blocked`

Reply:
Done